zynqmp.mk

```diff
@@ -27,75 +26,63 @@ else
 endif
 
 .PHONY: all
-all: petalinux-create petalinux-config petalinux-build petalinux-package
+all: create build package
 
-.PHONY: check-petalinux
-check-petalinux:
+.PHONY: check
+check:
 ifndef PETALINUX_VER
 	$(error You have to source Petalinux settings)
 endif
-ifneq ($(PETALINUX_VER),2018.2)
+ifneq ($(PETALINUX_VER),2020.2)
 	$(error This makefile only support Petalinux 2018.2)
```

Comment:
Petalinux version should be updated in the error message

Reply:
Right, let me address it in the next iteration